ocsci/tests/test_config.py

```diff
@@ -13,7 +12,9 @@ def reset_config(self):
 
     def test_defaults(self):
         ocsci.main.init_ocsci_conf()
-        config_sections = ocsci.config.to_dict().keys()
-        for section_name in config_sections:
-            section = getattr(ocsci.config, section_name)
-            assert section == getattr(ocs.defaults, section_name)
+        env_data = getattr(ocsci.config, 'ENV_DATA')
+        reporting_data = getattr(ocsci.config, 'REPORTING')
+        run_data = getattr(ocsci.config, 'RUN')
+        assert env_data['rook_image'] == 'rook/ceph:master'
+        assert reporting_data['email']['address'] == 'ocs-ci@redhat.com'
+        assert run_data['bin_dir'] == './bin'
```

Comment:
I think the existing test should be left alone, and a new test added if you want to verify specific values.

Reply:
The old assert where it tries to do "assert section == getattr(ocs.defaults, section_name)" is not valid since it would expect you to have values in default_config.yaml as well as define in defaults.py, The purpose of the PR was to remove the duplicates.